src/test/java/com/aerokube/selenoid/TestWindowCommands.java

```diff
@@ -47,9 +47,9 @@ public void testSwitchWindows() throws Exception {
             WebElement link = driver.findElement(By.id("test-link")); //This link opens new window
             link.click();
             await().until(() -> getDriver().getWindowHandles().size() == 2);
-            List<String> windowNames = getDriver().getWindowHandles().stream().map(wh -> {
+            List<String> windowNames = driver.getWindowHandles().stream().map(wh -> {
```

Comment:
What's the purpose of replacing `getDriver()` to `driver` here?

Reply:
Nothing really. Just to avoid a call to getDriver(). Actually I also missed to change it on the await command. Should I reuse driver or call getDriver each time?